frontend/javascripts/oxalis/view/components/command_palette.tsx

```diff
@@ -0,0 +1,198 @@
+
+  const props: TracingViewMenuProps = useSelector((state: OxalisState) => {
+    return {
+      restrictions: state.tracing.restrictions,
+      task: state.task,
+      annotationType: state.tracing.annotationType,
+      annotationId: state.tracing.annotationId,
+      activeUser: state.activeUser,
+      isAnnotationLockedByUser: state.tracing.isLockedByOwner,
+      annotationOwner: state.tracing.owner,
+      isDownloadModalOpen: state.uiInformation.showDownloadModal,
+      isRenderAnimationModalOpen: state.uiInformation.showRenderAnimationModal,
+      isShareModalOpen: state.uiInformation.showShareModal,
+      isMergeModalOpen: state.uiInformation.showMergeAnnotationModal,
+      isZarrPrivateLinksModalOpen: state.uiInformation.showZarrPrivateLinksModal,
+      isUserScriptsModalOpen: state.uiInformation.showAddScriptModal,
+    };
+  });
+
+  const { activeUser, restrictions } = props;
+
+  const getMenuActions = (isViewMode: boolean) => {
+    if (!isInTracingView) return [];
+    if (isViewMode) {
+      const { items } = getViewDatasetMenu(null);
+      return items;
+    }
+    const { menuItems } = getTracingViewModalsAndMenuItems(props, null);
+    return menuItems;
+  };
+
+  const getTabsAndSettingsMenuItems = () => {
+    if (!isInTracingView) return [];
+    const commands: Command[] = [];
+
+    (Object.keys(userConfig) as [keyof UserConfiguration]).forEach((key, counter) => {
+      if (typeof userConfig[key] === "boolean") {
+        commands.push({
+          id: counter,
+          name: `Toggle ${getPhraseFromCamelCaseString(key)}`,
+          command: () => Store.dispatch(updateUserSettingAction(key, !userConfig[key])),
+          color: "#5660ff",
+        });
+      }
+    });
+    return commands;
+  };
+
+  const getNavigationEntries = () => {
+    if (activeUser == null) return [];
+    const commands: Command[] = [];
+    const basicNavigationEntries = [
+      { name: "Tasks (Dashboard)", path: "/dashboard/tasks" },
+      { name: "Annotations", path: "/dashboard/annotations" },
+      { name: "Datasets", path: "/dashboard/datasets" },
+      { name: "Time Tracking", path: "/timetracking" },
+    ];
+
+    const adminMenu = getAdministrationSubMenu(false, activeUser);
+    const adminCommands: Array<{ name: string; path: string }> = [];
+    adminMenu?.children.map((entry: { key: string }) => {
+      adminCommands.push({ name: getLabelForPath(entry.key), path: entry.key });
+    });
+
+    const statisticsCommands = Utils.isUserAdminOrManager(activeUser)
+      ? [
+          {
+            path: "/reports/projectProgress",
+            name: "Project Progress",
+          },
+          {
+            path: "/reports/availableTasks",
+            name: "Available Tasks",
+          },
+        ]
+      : [];
+
+    const navigationEntries = [...basicNavigationEntries, ...adminCommands, ...statisticsCommands];
+
+    navigationEntries.forEach((entry, counter) => {
+      commands.push({
+        id: counter,
```

Comment:
how about adding a `category: "Navigate"` property here. will look like this: https://main--5c34dd1c22fa8200244c31dd.chromatic.com/?path=/story/command-palette--with-multiple-highlights
the same for settings-related stuff

Reply:
I really like the idea and I think the example looks nice. But I feel like this prop might have been removed, because the typechecker doesnt know it and its not rendered. a search within the repo also seemed to yield results in tests and demos https://github.com/search?q=repo%3Aasabaylus%2Freact-command-palette+category&type=code

Reply:
ok, then let's forget about it for now :)